docs/docs/getting-started/configuring-your-first-project.md

```diff
@@ -12,11 +12,22 @@ To do so, head to WebPageTest’s [Request API Key page](https://www.webpagetest
 
 This email contains your WebPageTest API Key that we will use in the next section.
 
+## Import data from WPT
+
+WPT service provides numerouse options for running your tests ( location, bandwidth ), so you **must** import this data into your setup. To do this, via admin interface:
+1. head to `<your Falco instance URL>/admin/django_celery_beat/periodictask/` 
+1. mark checkbox for `Get all wpt audit configurations: 0 1 * * * (m/h/d/dM/MY) UTC` 
+1. from combobox (dropdown list) choose to Run the task
+
+
+![Run import task](/docs/static/img/wpt-update.png)
+
+
 ## Creating your first project
 
 To create your first project, head to `<your Falco instance URL>/admin/` and connect to the admin interface. Then click on the `+ Add` button in the `Projects` section, and you should see the following form:
 
-![The form used to create a project](/img/create-project.png)
+![The form used to create a project](/docs/static/img/create-project.png)
```

Comment:
Are you sure about this?

When I try accessing https://getfal.co/img/create-project.png it works as intended, but if I go to https://getfal.co/docs/static/img/create-project.png it gives a Docusaurus 404.

Reply:
I was checking with Github's preview. Doesn't it mean that dir structure here and at getfal.co differs?

Reply:
My guess is that [Docusaurus](https://v2.docusaurus.io/), the docs framework we use for Falco, doesn’t exactly mimic the URL structure with the file structure with regards to images / static assets.

Reply:
well, i cannot check it in advance, so no problem from my side to keep url as it was

Reply:
Great then! Thanks!